omnigibson/prims/cloth_prim.py

```diff
@@ -72,6 +74,9 @@ def __init__(
         )
 
     def _post_load(self):
+        # Cache the original scale of the visual mesh
```

Comment:
maybe add additional comment why we want to cache the original scale in the first place?